jest.config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
```

Comment:
This is a little annoying. Jest doesn't seem to like being set up to run multiple sub-folders' tests, so there's some inconsistency. Each project has its own Jest dependency but relies on this shared config, and since the `rootDir` is the root of the monorepo, running `jest` in a workspace will run all tests for all workspaces.

There doesn't appear to be a way to solve that in config without duplicating a ton of configuration (anywhere `<rootDir>` is used), so each workspace is manually scoping its test run to only that workspace's files